css/main.css

```diff
@@ -1,73 +1,83 @@
 body {
-    background-image: url("../img/tile.jpg");
-    background-color: rgba(255, 255, 255, 1);
-    font-family: 'EB Garamond', serif;
-    font-size: 250%;
+  background-color: #e1e1e1;
+  font-family: 'Roboto', serif;
+  font-weight: 300;
+  font-size: 200%;
+  max-width: 1200px;
```

Comment:
We need to build this into an actual panel, really, to make sure a lot of these sizes are correct